data/language/en-GB.yml

```diff
@@ -2428,9 +2428,9 @@ strings:
   2373: "{COLOUR WINDOW_2}Music volume:"
   2374: Show planned AI track/road as ghosts
   2375: "{SMALLFONT}{COLOUR BLACK}This shows the routes planned by AI companies as regular ghost elements."
-  2376: "{MOVE_X 14}{COLOUR BLACK}Title"
-  2377: "{MOVE_X 14}{COLOUR BLACK}Title ▲"
-  2378: "{MOVE_X 14}{COLOUR BLACK}Title ▼"
+  2376: "{MOVE_X 15}{COLOUR BLACK}Title"
+  2377: "{MOVE_X 15}{COLOUR BLACK}Title ▲"
+  2378: "{MOVE_X 15}{COLOUR BLACK}Title ▼"
```

Comment:
Are you sure about this? I realise the checkbox-title offset is 15px, but I thought I was compensating for the border here.

Reply:
Alright, let's leave it at this then.